index.d.ts

```diff
@@ -89,6 +89,34 @@ declare module 'binance-api-node' {
     locked: string
   }
 
+  export type booleanString = 'true' | 'false'
+
+  export interface positionAmount {
+    amount: string
+    amountInBTC: string
+    amountInUSDT: string
+    asset: string
+  }
+
+  export interface LendingAccount {
+    positionAmountVos: positionAmount[]
+    totalAmountInBTC: string
+    totalAmountInUSDT: string
+    totalFixedAmountInBTC: string
+    totalFixedAmountInUSDT: string
+    totalFlexibleInBTC: string
+    totalFlexibleInUSDT: string
+  }
+
+  export interface FundingWallet {
+    asset: string
+    free: string    // avalible balance
```

Comment:
little comment typo

Reply:
> little comment typo

haha, it was just copied from the official documentation (with the typo)